### PR TITLE
refactor!: remove `_id` suffix for entity ids in public API

### PR DESCRIPTION
**Pull request by mversic**
## Description

Remove `_id` suffix from tokens, config files and data model

### Linked issue

Closes #{issue_number} 

### Checklist

- [ ] I've read `CONTRIBUTING.md`
- [ ] I've used the standard signed-off commit format (or will squash just before merging)
- [ ] All applicable CI checks pass (or I promised to make them pass later)
- [ ] (optional) I've written unit tests for the code changes
- [ ] I replied to all comments after code review, marking all implemented changes with thumbs up

### CODE REVIEW

**Review by nxsaken — APPROVED** [review]

File: scripts/test_env.py [hunk 1]
```diff
@@ -247,7 +247,7 @@ def setup(args: argparse.Namespace):
 
 def cleanup(out_dir: pathlib.Path):
     logging.info("Killing peer processes...")
-    subprocess.run(["pkill", "-9", "irohad"])
+    subprocess.run(["pkill", "-9", "iroha"])
```
Comment (0x009922) [inline scripts/test_env.py:250]: Is it right?

File: smart_contract/executor/src/default.rs [hunk 2]
```diff
@@ -251,7 +251,7 @@ pub mod domain {
         authority: &AccountId,
         isi: &SetKeyValue<Domain>,
     ) {
-        let domain_id = isi.object_id();
+        let domain_id = isi.object();
```
Comment (0x009922) [inline smart_contract/executor/src/default.rs:254]: Although IMO it's totally fine to not rename local variables.

```suggestion
        let domain = isi.object();
```

**Review by 0x009922 — APPROVED** [review]

File: scripts/test_env.py (hunk 1 above)
Comment (mversic) [reply scripts/test_env.py:250]: it is because processes are named `iroha0` etc

File: smart_contract/executor/src/default.rs (hunk 2 above)
Comment (mversic) [reply smart_contract/executor/src/default.rs:254]: ah, I didn't go renaming local variables because that would sometimes conflict with entity names

**Pull request merged**